src/GuessMessage.js

```diff
@@ -1,27 +1,27 @@
-function getGuessMessage(guess, numberToGuess) {
-  const distanceApart = Math.abs(numberToGuess - guess);
-  if (guess < numberToGuess) {
-    if (distanceApart > 10) {
-      return "Your guess was well below the number.";
+const getGuessMessage = (guess, numberToGuess) => {
+    const distanceApart = Math.abs(numberToGuess - guess);
+    if (guess < numberToGuess) {
+        if (distanceApart > 10) {
+            return "Your guess was well below the number.";
+        } else {
+            return "Your guess was a little below the number.";
+        }
     } else {
-      return "Your guess was a little below the number.";
+        if (distanceApart > 10) {
+            return "Your guess was well above the number.";
+        } else {
+            return "Your guess was a little above the number.";
+        }
     }
-  } else {
-    if (distanceApart > 10) {
-      return "Your guess was well above the number.";
-    } else {
-      return "Your guess was a little above the number.";
-    }
-  }
-}
+};
```

Comment:
Going beyond the assignment (not required):
This function has some duplicated logic where its checking if the distance is greater than 10 multiple times. How might you simplify this duplicate logic?

Reply:
I'd probably do something like this

```
const getGuessMessage = (guess, numberToGuess) => {
    const distanceApart = Math.abs(numberToGuess - guess);
    let message;

    if (guess < numberToGuess) {
        message = "Your guess was a little below the number.";
    } else {
        message = "Your guess was a little above the number.";
    }

    if (distanceApart > 10) {
        message = message.replace("a little", "well")
    }

    return message;
};
```